bin/posthog

```diff
@@ -27,21 +27,21 @@ command :capture do |c|
   c.option '--event=<event>', String, 'The event name to send with the event'
   c.option '--properties=<properties>', 'The properties to send (JSON-encoded)'
 
-  c.action do |args, options|
+  c.action do |_args, options|
     posthog =
       PostHog::Client.new(
         {
-          api_key: options.api_key,
-          api_host: options.api_host,
-          on_error: Proc.new { |status, msg| print msg }
+          :api_key => options.api_key,
```

Comment:
Nah, hate this one, can we update the rule? Colon syntax is the modern one, rocket (`=>`) is antiquated